pkg/builder3/openapi_test.go

```diff
@@ -90,10 +91,11 @@ func (_ TestInput) OpenAPIDefinition() *openapi.OpenAPIDefinition {
 		},
 	}
 	schema.Extensions = spec.Extensions{"x-test": "test"}
-	return &openapi.OpenAPIDefinition{
+	def := openapi.EmbedOpenAPIDefinitionIntoV2Extension(openapi.OpenAPIDefinition{
 		Schema:       schema,
 		Dependencies: []string{},
-	}
+	}, openapi.OpenAPIDefinition{})
```

Comment:
If I'm reading correctly, this would embed an empty v2extension in the test schema, and since the test is unaffected, it proves that the schema is dropped, is that correct?

Reply:
Yes, I am adding comments to clarify this.